content/servers.json5

```diff
@@ -49,10 +49,10 @@ links (array): objects representing game join links:
 		{
 			name: '/TG/Station 13',
 			logo: 'tgstation.png',
-			tags: ['experienced', 'lightrp'],
+			tags: ['experienced', 'lightrp', 'mediumrp', 'casual', 'expert'],
 			website: 'https://tgstation13.org',
 			description: [
-				'/tg/station 13 spawned from the /tg/ (Traditional Games) section of 4chan, an imageboard primarily dedicated to the discussion of board games and tabletop RPGs. Using Goonstation’s publically released codebase as a jumping off point, /tg/station quickly branched off – implementing plenty of features and quality of life improvements to the game, causing it to soar in popularity. Sybil and Basil operate coderbus’s standard /tg/station13 code. A third server called Artyom operates a seperate branch of code, referred to as NTstation. The gameplay style is a comfortable middle-ground between Goonstation and Baystation 12, and a newbie shouldn’t have too many problems jumping in.',
+				'/tg/Station 13 (from the /tg/ (Traditional Games) section of 4chan), quickly became the primary open source upstream after goonstation went closed source, making them the closest thing to \'vanilla\' SS13. /tg/station has multiple servers at multiple locations and roleplaying levels, and annoyingly strides to fit their server perfectly between rp levels. Sybil and Basil in the US and Terry in the EU operate As middleground Low-RP/Medium-RP servers. A third server called Manuel operates as as their middleground Medium-RP/High-RP server, with a EU version in the works since the dawn of man. The gameplay style is a comfortable middle-ground between Goonstation and Baystation 12, and a newbie shouldn’t have too many problems jumping in. They host yearly events that bring in members from the entire ss13 community',
```

Comment:
```suggestion
				'/tg/Station 13 (from the /tg/ (Traditional Games) section of 4chan), quickly became the primary open source upstream after goonstation went closed source, making them the closest thing to \'vanilla\' SS13. /tg/station has multiple servers at multiple locations and roleplaying levels, and annoyingly strides to fit their server perfectly between rp levels. Sybil and Basil in the US and Terry in the EU operate as middleground Low-RP/Medium-RP servers. A third server called Manuel operates as their middleground Medium-RP/High-RP server, with an EU version in the works since the dawn of man. The gameplay style is a comfortable middle-ground between Goonstation and Baystation 12, and a newbie shouldn’t have too many problems jumping in. They host yearly events that bring in members from the entire ss13 community',
```